examples/create_mask.rs

```diff
@@ -0,0 +1,178 @@
+//! # Create Mask
+//! Reference: [opencv/samples/cpp/create_mask.cpp](https://github.com/opencv/opencv/blob/4.x/samples/cpp/create_mask.cpp)
+
+use opencv::{
+	core::{bitwise_and, find_file, CommandLineParser, Point, Scalar, CV_8UC1, CV_8UC3},
+	highgui::{self, *},
+	imgcodecs::{imread, IMREAD_COLOR},
+	imgproc,
+	prelude::*,
+};
+
+use std::{
+	env, process,
+	sync::{
+		atomic::{self, AtomicBool},
+		Arc, Mutex,
+	},
+};
+
+const SOURCE_WINDOW: &str = "Source image";
+
+fn main() {
+	let args: Vec<String> = env::args().collect();
+	let (argc, argv) = (args.len() as i32, args.iter().map(|s| s.as_str()).collect::<Vec<&str>>());
+
+	let mut parser = CommandLineParser::new(argc, &argv, "{@input | lena.jpg | input image}").unwrap();
+	parser.about("This program demonstrates using mouse events\n").unwrap();
+	parser.print_message().unwrap();
+	println!(
+		"\n\tleft mouse button - set a point to create mask shape\n\
+            \tright mouse button - create mask from points\n\
+            \tmiddle mouse button - reset"
+	);
+
+	// HACK: replace with parser.get<String>("@input")
+	let input_image = "lena.jpg";
+	let input_image_path = find_file(input_image, true, false).expect("Cannot find input image");
```

Comment:
@twistedfall When I working on this example, I noticed that there is a `get` method used in the CommandLineParser ([opencv C++/create_mask.cpp](https://github.com/opencv/opencv/blob/131dab774c386217d323c00248b0276bd4033dda/samples/cpp/create_mask.cpp#L102)),  but I couldn't find it in this project. Can you please provide some suggestions on how to resolve this issue? Thank you!

Reply:
Those generic methods need an extra push to be generated. Because there is no way to automatically translate the generics between C++ and Rust we need to specify the necessary specializations manually here: https://github.com/twistedfall/opencv-rust/blob/master/binding-generator/src/settings.rs#L784 I'll try to create a couple of useful specializations for this function in the next release.